yail/core.py

```diff
@@ -34,14 +34,13 @@ def generator(it):
     Examples:
 
         >>> generator(range(5))  # doctest: +ELLIPSIS
-        <generator object generator at 0x...>
+        <generator object ...<genexpr> at 0x...>
```

Comment:
Apparently on the CIs this is `generator.<locals>.<genexpr>` as seen in this [build]( https://travis-ci.org/jakirkham/yail/jobs/169385338#L520 ). However, locally it is just `<genexpr>`. Not sure why the discrepancy. However, adding an `...` works for both cases.